mapie/multi_label_classification.py

```diff
@@ -712,7 +770,7 @@ def predict(
 
         Parameters
         ----------
-        X : ArrayLike of shape (n_samples, n_features)
+        X: ArrayLike of shape (n_samples, n_features)
             Test data.
         method : Optional[str]
```

Comment:
UPD: this part of the docstring should be removed (and add a newline).

Reply:
Why do you want to remove the docString about parameter X?

Reply:
NVM you meant method argument